app/controllers/discover.js

```diff
@@ -240,6 +240,16 @@ export default ApplicationController.extend({
             this.set(type, newValue);
         },
 
+        removeFilter(type, filterValue) {
+            let currentValue = getSplitParams(this.get(type));
+            currentValue = currentValue ? currentValue : [];
```

Comment:
These two lines could be condensed to `let currentValue = getSplitParams(this.get(type)) || [];`, which would be nicer.